_plugins/to_platforms.rb

```diff
@@ -0,0 +1,39 @@
+module Jekyll
+  class ToPlatforms < Jekyll::Generator
+    def generate(site)
+      platforms = site.data['platforms']
```

Comment:
super nitpicky but why single quotes instead of double quotes?

Reply:
They are the sam(ish) tbh, no rubocop rules in here hehe